### ISSUE TITLE
System cursor is not shown in fullscreen

### ISSUE THREAD

**Issue by Poobslag** [26F-Studio/Techmino#1027]
In Windows 10, with "Use System Cursor" enabled no cursor is visible at all in fullscreen. Moving the mouse around still highlights game elements so you can kind of guess approximately where your cursor is, but the system cursor is not shown.

Unchecking "Use System Cursor" is an acceptable workaround, and the game's cursor is shown appropriately. Disabling fullscreen mode with "F11" also fixes the problem.

Steps to reproduce:

1. Launch Techmino. Disable fullscreen mode (F11).
2. In the 'Game Settings' menu, enable 'Use System Cursor'
3. Enable Fullscreen mode (F11).  The system cursor is hidden.

**Comment by shoucandanghehe** [comment]
Hi @Poobslag , can you provide more information? If you don't reply, I will close this issue. (reopen this if you have the same problem)

**Comment by Poobslag** [comment]
> Hi @Poobslag , can you provide more information? If you don't reply, I will close this issue. (reopen this if you have the same problem)

I have provided my operating system and detailed steps for reproducing the error. What additional information would you like?

**Comment by Poobslag** [comment]
> We're not sure what's going on. I'm on Windows 10 and could not reproduce this issue. Did you install a custom cursor or something? If not, it may also be something to do with your device drivers.

No, I do not have a custom cursor, but I found one helpful data point..

![image](https://github.com/26F-Studio/Techmino/assets/5902413/7b77e90d-4526-44bc-b600-617a541907d5)

After some further experimentation, I've found the bug only occurs if "Display pointer trails" is enabled in Windows 10. If I disable this setting, the cursor appears normally. Can you reproduce the issue with this mouse setting enabled?

**Comment by shoucandanghehe** [comment]
> Can you reproduce the issue with this mouse setting enabled?

No, I can't reproduce the bug even if I enable Display pointer trails
But I using Windows 11🤔
